fbw-a32nx/src/wasm/systems/a320_systems/src/pneumatic.rs

```diff
@@ -1359,41 +1739,45 @@ impl CrossBleedValve {
         container_one: &mut impl PneumaticContainer,
         container_two: &mut impl PneumaticContainer,
     ) {
-        if !self.is_powered_for_manual_control && !self.is_powered_for_automatic_control {
-            self.set_open_amount_from_pressure_difference(
-                container_one.pressure() - container_two.pressure(),
-            )
-        }
+        self.open_amount = if self.target_open_amount > self.open_amount {
+            self.target_open_amount
+                .min(self.open_amount + context.delta_as_secs_f64() * self.valve_speed)
```

Comment:
Extract calculation into separate variable to avoid code duplication (the `time * valve_speed` part).